src/diagonal.works/b6/test/data.go

```diff
@@ -14,12 +14,34 @@ const (
 	CamdenPBF        = "camden.osm.pbf"
 )
 
+
+func findModuleRoot(dir string) (roots string) {
```

Comment:
This was taken directly from here - https://github.com/golang/go/blob/9e3b1d53a012e98cfd02de2de8b1bd53522464d4/src/cmd/go/internal/modload/init.go#L1504